README.mdown

```diff
@@ -32,3 +32,20 @@ To run tests do:
     python2 bookshelf/api_v2/test_ec2.py
     python2 bookshelf/api_v2/test_rackspace.py
 
+There are also tests for the api_v3 that spin up GCE, Rackspace, and EC2
+instances. These tests require credentials to access GCE, Rackspace, and EC2.
+In an effort to attempt to reduce the amount of work to set up the
+configuration, they build on the existing acceptance.yml as described at:
+http://doc-dev.clusterhq.com/gettinginvolved/appendix.html#acceptance-testing-configuration.
+
```

Comment:
this one is really bugging me,
The code in this repo doesn't use anything from the flocker codebase. The idea is that it can be consumed horizontally to build and mange all clusterHQ microservices and fat services, without being constrained to changes happening in the flocker code base.
Considering that you already output what the yaml config should be if someone tries to run this without a valid config, then I would prefer if you remove L37-39.
pleeeeease